### PR TITLE
test(menu): added context menu test

### LINKED ISSUE #1667: Enhance Test Coverage for `@yamada-ui/menu`
package: @yamada-ui/menu

### Description
`@yamada-ui/menu` has a test coverage of less than 90%. We will enhance it to above 90%.

### Target files
- [packages/components/menu/src/context-menu-trigger.tsx](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/menu/src/context-menu-trigger.tsx)
  - [L15](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/menu/src/context-menu-trigger.tsx#L15)
- [packages/components/menu/src/context-menu.tsx](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/menu/src/context-menu.tsx)
  - [L33](https://app.codecov.io/gh/yamada-ui/yamada-ui/blob/main/packages/components/menu/src/context-menu.tsx#L33)

### Problem Statement/Justification
Testing is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.

### PR DESCRIPTION
**Pull request by Kei-Ta**
Closes #1667

## Description

Added context menu tests for enhance test coverage.

## Current behavior (updates)

NIL

## New behavior

NIL

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/components/menu/tests/menu.test.tsx [hunk 1]
```diff
@@ -373,4 +374,81 @@ describe("<Menu />", () => {
...
+        <MenuList data-testid="MenuList">
+          <MenuItem data-testid="MenuItem">Undo</MenuItem>
+          <MenuItem>Redo</MenuItem>
+        </MenuList>
+      </ContextMenu>,
+    )
+    expect(screen.getByTestId("ContextMenuTrigger")).toBeInTheDocument()
+    expect(screen.getByText("Right click here")).toBeInTheDocument()
+    expect(screen.getByTestId("MenuList")).toBeInTheDocument()
+    expect(screen.getByTestId("MenuItem")).toBeInTheDocument()
```
Comment (108yen) [inline packages/components/menu/tests/menu.test.tsx:424]: I think this is sufficient.
```ts
expect(screen.getByText("Right click here")).toBeInTheDocument()
```

File: packages/components/menu/tests/menu.test.tsx [hunk 2]
```diff
@@ -373,4 +374,81 @@ describe("<Menu />", () => {
...
+        >
+          Right click here
+        </ContextMenuTrigger>
+        <MenuList>
+          <MenuItem>Undo</MenuItem>
+          <MenuItem>Redo</MenuItem>
+        </MenuList>
+      </ContextMenu>,
+    )
+    const contextMenuTrigger = screen.getByTestId("ContextMenuTrigger")
```
Comment (108yen) [inline packages/components/menu/tests/menu.test.tsx:447]: Can the following codes be substituted?
```
const contextMenuTrigger = screen.getByText("Right click here")
```

**Review by 108yen — CHANGES_REQUESTED** [review]
Avoid using `getByTestId` if at all possible.

File: packages/components/menu/tests/menu.test.tsx (hunk 2 above)
Comment (Kei-Ta) [reply packages/components/menu/tests/menu.test.tsx]: @108yen 
Thank you for your reviews.
I fixed it.
I have one question. Why should you avoid using getByTestId?

Comment (108yen) [reply packages/components/menu/tests/menu.test.tsx]: @Kei-Ta 
Thank you for fixing it!
`getByTestId` is a last resort. `fireEvent` should also be used as little as possible.
Read this article.
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library

**Review by 108yen — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**